edgelet/iotedge/src/check/checks/storage_mounted_from_host.rs

```diff
@@ -0,0 +1,132 @@
+use std::path::Path;
+
+use failure::{self, Context, ResultExt};
+use regex::Regex;
+
+use crate::check::{checker::Checker, Check, CheckResult};
+
+#[derive(Default, serde_derive::Serialize)]
+pub(crate) struct EdgeAgentStorageMounted {}
+impl Checker for EdgeAgentStorageMounted {
+    fn id(&self) -> &'static str {
+        "edge-agent-storage-mounted-from-host"
+    }
+    fn description(&self) -> &'static str {
+        // Note: Keep in sync with Microsoft.Azure.Devices.Edge.Agent.Service.Program.GetStoragePath
```

Comment:
Also update the two `GetStoragePath` functions - they each have a comment pointing to `edgelet/iotedge/src/check/mod.rs` that needs to be updated to `edgelet/iotedge/src/check/checks/storage_mounted_from_host.rs`